pkgs/servers/windmill/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, callPackage
+, pkg-config
+, openssl
+, python3
+, deno
+, librusty_v8 ? callPackage ../../development/web/deno/librusty_v8.nix { }
+, swagger-cli
+, nsjail
+, lld
+, makeWrapper
+, rustfmt
+, go
+, bash
+}:
+
+let
+  pname = "windmill";
+  version = "1.56.1";
+  fullSrc = fetchFromGitHub {
+    owner = "windmill-labs";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-M2/tTZFV2YjrykLVELjFE9iQhzcrQtRCIXZ2EpTpuE8=";
+  };
+
+  frontend-build = callPackage ./ui.nix { };
+
+  # a Cargo dep requires nightly rustfmt
```

Comment:
Should we use nightly things, especially nightly Rust things?

Reply:
We already use it - https://github.com/NixOS/nixpkgs/blob/54c0e4c4bb095ff374f7a29879a92aa2954e2dd0/pkgs/development/tools/rust/bindgen/unwrapped.nix#L7

Reply:
Which dep exactly? Can we patch this out?

Reply:
Added a comment pointing out which dep exactly. I don't think patching it will be straighforward as nightly features might actually be used.